### PR TITLE
Migrate `testing/requestErrors`

### PR DESCRIPTION
**Pull request by mkondratek**
# Based on PR #6289

## Full chain of PRs as of 2024-12-10

* PR #6293:
  `mkondratek/migrate-api/testing/requestErrors` ➔ `pkukielka/fix-polly-recordings-v5`
* PR #6289:
  `pkukielka/fix-polly-recordings-v5` ➔ `main`

## Test plan
- All migrated endpoints have been verified with debugger (and reviewed in the trace log).
- Autocomplete works, chat works.

### CODE REVIEW

File: jetbrains/src/integrationTest/kotlin/com/sourcegraph/cody/util/CodyIntegrationTextFixture.kt [hunk 1]
```diff
@@ -143,6 +148,29 @@ open class CodyIntegrationTextFixture : BasePlatformTestCase(), LensListener {
     assertTrue("Action should be visible", presentation.isVisible)
   }
 
+  private fun isAuthenticated() {
```
Comment (pkukielka) [inline jetbrains/src/integrationTest/kotlin/com/sourcegraph/cody/util/CodyIntegrationTextFixture.kt:151]: That looks wrong. After my recent changes if init succeeded, correct auth state should be returned out of it. There should be no need for active waiting. I will give it a try and if needed change the PR.

Comment (mkondratek) [reply jetbrains/src/integrationTest/kotlin/com/sourcegraph/cody/util/CodyIntegrationTextFixture.kt]: removed

Comment (pkukielka) [reply jetbrains/src/integrationTest/kotlin/com/sourcegraph/cody/util/CodyIntegrationTextFixture.kt]: Actually that auth check was showing real issue.
But I can push a fix for it on a separate PR, I have it ready.
Let's merge that first.

**Review by pkukielka — APPROVED** [review]
LGTM

**Pull request merged**